src/cfnlint/core.py

```diff
@@ -156,19 +164,19 @@ def get_args_filenames(cli_args):
         sys.exit(0)
 
     if not sys.stdin.isatty() and not config.templates:
-        return(config, [None], formatter)
+        return (config, [None], formatter)
```

Comment:
Not sure if this change is made by formatter (along with others below). Since there are no logical changes, let's keep style consistent across different files.

Reply:
Done!